cli/index.ts

```diff
@@ -169,7 +176,10 @@ const builtYargs = createYargsCli({
               warehouse: argv.warehouse,
               gcloudProjectId: argv["gcloud-project-id"]
             },
-            argv["skip-install"]
+            {
+              skipInstall: argv["skip-install"],
+              includeSchedules: argv['include-schedules']
```

Comment:
i'm surprised prettier isn't replacing these `'` with `"` (seems like it should be consistent with the previous line)